README.md

```diff
@@ -140,20 +148,27 @@ The convention for dbNames are "openmrs_[some name]".
 ```
 $ mvn openmrs-sdk:setup -DserverId=[serverId] -Ddistro=org.openmrs.module:mirebalais:1.2-SNAPSHOT
 ```
+or
+```
+$ invoke setup
+```
 
 * When prompted, select the port you'd like to run tomcat on
 
 * When prompted, set the port to debug on (standard is 1044)
 
-* Select which database you'd like to use
+* For database selection, select either the option to use locally-installed MySQL, or to use a MySQL docker container.
+  Option "3. Use an existing docker container," does not seem to work at the time of this writing. Your
+  `serverId` must only contain
+  [MySQL Permitted Characters in Unquoted Identifiers](https://dev.mysql.com/doc/refman/8.0/en/identifiers.html).
 
-* If you are connecting to a MySQL 5.6 instance running on your local machine, specifc the URI, and a username and password to connect to the DB
+* If you are connecting to a MySQL 5.6 instance running on your local machine (and not using Invoke) specify the URI, and a username and password to connect to the DB
```

Comment:
In general I think we need to assume that they are _either_ following these instructions, _or_ using invoke, and that we should have a separate section below that indicates what invoke commands should be run that correspond with this process